README.rst

```diff
@@ -58,12 +58,164 @@ Python:
   provides an abstraction of NI-DAQmx in the ``nidaqmx`` module, which collides
   with this package's module name.
 
+Getting Started
+===============
+
+In order to use the **nidaqmx** package, you must have at least one DAQ device installed on your
+machine. X series DAQ device (eg: PCIe-6363 or USB-6351) is recommended, physical or
+simulated devices are supported. You can use the **NI MAX** or **NI Hardware Configuration Utility**
```

Comment:
"NI MAX", not "the NI MAX"

Apparently "Hardware Configuration Utility" is the same way: https://www.ni.com/docs/en-US/bundle/hardwareconfigurationutility/page/definition.html

```suggestion
simulated devices are supported. You can use **NI MAX** or **NI Hardware Configuration Utility**
```